### PR TITLE
Feature/verification screen

### PR DESCRIPTION
**Pull request by alissonfpmorais**
This PR adds the verification screen.

Run this branch and check the work in the following URL:
http://localhost:3000/community/PUL/objective/1/action/1/claim/1/verification

Changes:
- [x] extend tailwind default behaviour;
- [x] add new route to verification screen;
- [x] show verification (verification screen);
- [x] perform request to verify the claim;
- [x] add translations;
- [x] add queries to retrieve verification from API.

Some screenshots:
Screenshot from 2019-09-20 16-08-39
Screenshot from 2019-09-20 16-16-54
Screenshot from 2019-09-20 16-09-25
Screenshot from 2019-09-20 16-09-44
Screenshot from 2019-09-20 16-09-58
Screenshot from 2019-09-20 16-16-39

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -69,7 +69,7 @@
 /* Button */
 
 .button {
-    @apply uppercase font-sans text-sm leading-none py-2 px-6 w-40 h-10;
+    @apply uppercase font-sans text-sm text-center leading-none py-2 px-6 w-40 h-10;
```
Comment (zacck) [inline src/styles/main.css]: @alissonfpmorais  awesome!!! @lucca65  this fixes the explore button text in English 
Screen Shot 2019-09-23 at 08 06 31

**Comment by alissonfpmorais** [comment]
> @alissonfpmorais This looks and works awesome! Awesome work dude!!
> 
> I noticed a slight issue when one brings up the confirmation modal, sometimes a part of the modal is not visible as in the print below, when this happens only the modal responds to scrolling which can be very confusing .. would it be possible to make the whole grey area respond to scrolling
> 
> Screen Shot 2019-09-23 at 08 01 13

Hey @zacck thanks for the check!

I've sent a fix to that bug right now, could you please check if is it still happening?

File: tailwind.config.js [hunk 2]
```diff
@@ -161,6 +164,18 @@ module.exports = {
...
+    function ({ addUtilities }) {
+      const newUtilities = {
+        '.inset-center': {
+          top: '50%',
+          left: '50%',
+          transform: 'translate(-50%, -50%)'
+        }
+      }
+
+      addUtilities(newUtilities, ['responsive'])
```
[the remark this answers is not in the document]
Comment (lucca65) [reply tailwind.config.js]: about the modal not being on centre of the page, here is an example from the shop:

https://www.figma.com/file/GyRvSEQNS91DOLgMIk3u77/shop---desk

File: src/elm/Route.elm [hunk 3]
```diff
@@ -26,6 +25,7 @@ type Route
     | NewCommunity
     | EditCommunity Symbol
     | NewAction Symbol String
+    | VerifyClaim String String String String
```
Comment (lucca65) [inline src/elm/Route.elm]: this should be `VerifyClaim Symbol String String String`

File: src/elm/Asset/Icon.elm [hunk 4]
```diff
@@ -195,6 +195,63 @@ close =
...
+        ]
+        [ path
+            [ fillRule "evenodd"
+            , clipRule "evenodd"
+            , d "M13.3238 25C12.7916 24.9995 12.2815 24.7955 11.9054 24.4328L2.5387 15.4087C1.80178 14.6467 1.82353 13.4594 2.58788 12.723C3.35224 11.9866 4.58463 11.9657 5.37547 12.6756L13.3238 20.3332L26.6245 7.519C27.4154 6.80903 28.6478 6.82998 29.4121 7.56638C30.1765 8.30278 30.1982 9.4901 29.4613 10.252L14.7422 24.4328C14.3661 24.7955 13.8559 24.9995 13.3238 25Z"
+            ]
+            []
+        ]
+
+
```
[the remark this answers is not in the document]
Comment (alissonfpmorais) [reply src/elm/Asset/Icon.elm]: Using now the new `Icons` module

File: src/elm/Page/Community/VerifyClaim.elm [hunk 5]
```diff
@@ -0,0 +1,659 @@
...
+                            [ viewHeader shared.endpoints.ipfs verification.logo verification.name
+                            , viewStatus shared.translations verification.symbol verification.verifierReward verification.status
+                            , viewInfo shared.translations verification
+                            , viewAction shared.translations verification
+                            ]
+                        ]
+
+                Nothing ->
+                    -- TODO: add screen case verification does not exist
+                    Debug.todo ""
```
[the remark this answers is not in the document]
Comment (alissonfpmorais) [reply src/elm/Page/Community/VerifyClaim.elm]: Update done

File: src/elm/Page/Community/VerifyClaim.elm [hunk 6]
```diff
@@ -0,0 +1,659 @@
...
+                        [ div
+                            [ class "flex justify-between items-center" ]
+                            [ p
+                                [ class "text-heading font-bold" ]
+                                [ text (t "verify_claim.modal.title") ]
+                            , button
+                                [ class "w-8 h-8"
+                                , onClick (ClickedClose verification)
+                                ]
+                                [ Asset.Icon.close "text-gray-900 fill-current" ]
```
[the remark this answers is not in the document]
Comment (alissonfpmorais) [reply src/elm/Page/Community/VerifyClaim.elm]: Same as above, using it now

File: src/elm/Page/Community/VerifyClaim.elm [hunk 7]
```diff
@@ -0,0 +1,659 @@
...
+                                  , name = "verifyclaim"
+                                  , authorization =
+                                        { actor = accountName
+                                        , permissionName = Eos.samplePermission
+                                        }
+                                  , data = encodeVerification model.claimId accountName vote
+                                  }
+                                ]
+                            }
+                    }
```
[the remark this answers is not in the document]
Comment (alissonfpmorais) [reply src/elm/Page/Community/VerifyClaim.elm]: Done, auth is now check and asked if needed

File: tailwind.config.js [hunk 8]
```diff
@@ -101,6 +101,9 @@ module.exports = {
         'title-h1': '3rem',
         'hero': '3.375rem'
       },
+      margin: {
+        '28': '7rem'
+      },
```
[the remark this answers is not in the document]
Comment (alissonfpmorais) [reply tailwind.config.js]: Removed, changed to `mt-24`

File: tailwind.config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (alissonfpmorais) [reply tailwind.config.js]: Remove and change the modal's position on the screen

File: src/elm/Route.elm (hunk 3 above)
Comment (alissonfpmorais) [reply src/elm/Route.elm]: Updated to use `Symbol` instead of `String`

**Comment by zacck** [comment]
@alissonfpmorais  yes that's fixed now 👍

File: src/elm/Page/Dashboard.elm [hunk 9]
```diff
@@ -200,8 +263,8 @@ viewActivity url activity =
         ]
 
 
-viewNoActivity : List (Html Msg) -> Html Msg
-viewNoActivity elements =
+viewNoVerification : List (Html Msg) -> Html Msg
+viewNoVerification elements =
```
Comment (lucca65) [inline src/elm/Page/Dashboard.elm]: 😌

File: tailwind.config.js (hunk 8 above)
[the remark this answers is not in the document]
Comment (lucca65) [reply tailwind.config.js]: nice! please also remove this from the config!

File: tailwind.config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (lucca65) [reply tailwind.config.js]: no need for this config anymore too! we can remove this!

**Pull request merged**